src/Spout/Writer/Common/Entity/Cell.php

```diff
@@ -81,6 +100,25 @@ public function getValue()
     }
 
     /**
+     * @param Style $style
+     */
+    public function setStyle(Style $style)
+    {
+        $this->style = $style;
+    }
+
+    /**
+     * @return Style
+     */
+    public function getStyle()
+    {
+        if (!isset($this->style)) {
```

Comment:
It looks wrong to have `getStyle` call `setStyle`. What about setting `$this->style` to `new Style()` when `setStyle(null)` is called (like in the constructor)? This way your getter will simply be a return statement